controllers/topics.controller.js

```diff
@@ -0,0 +1,9 @@
+const { fetchAllTopics } = require('../models/topics.models');
+
+exports.getTopics = (req, res, next) => {
+  return fetchAllTopics()
+    .then((topics) => {
+      res.status(200).send(topics);
```

Comment:
we should send back data on a meaningful key - e.g. the user should be able to access the topics on body.topics